File: weather/src/main/java/com/nerdery/pkarels/weather/data/model/HourlyResponse.java
package com.nerdery.pkarels.weather.data.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class HourlyResponse {
    @SerializedName("data")
    List<ForecastCondition> hours;

    public HourlyResponse(List<ForecastCondition> hours) {
        this.hours = hours;
    }

    /**
     * Ordered List of Hourly {@link ForecastCondition}
     * @return List of {@link ForecastCondition}
     */
    public List<ForecastCondition> getHours() {
        return hours;
    }
}

File: weather/src/main/java/com/nerdery/pkarels/weather/model/HourlyResponse.kt
package com.nerdery.pkarels.weather.model

import com.google.gson.annotations.SerializedName

class HourlyResponse(hours: List<ForecastCondition>) {
    /**
     * Ordered List of Hourly [ForecastCondition]
     * @return List of [ForecastCondition]
     */
    @SerializedName("data")
    var hours: List<ForecastCondition>
        internal set

    init {
        this.hours = hours
    }
}
